channeldb/db.go

```diff
@@ -263,3 +296,123 @@ func (d *DB) FetchAllChannels() ([]*OpenChannel, error) {
 
 	return channels, err
 }
+
+// SyncVersions function is used for safe db version synchronization. It applies
+// migration functions to the current database and recovers the previous
+// state of db if at least one error/panic appeared during migration.
+func (d *DB) SyncVersions(versions []Version) (returnErr error) {
```

Comment:
Usage of named returns for functions is purposefully avoided across the project. 

Can you modify this function to use regular returns? Thanks